### DISCUSSION TITLE
Playlist: Make playlist for different time ranges for a dashboard.

### DISCUSSION THREAD

**Discussion by utkarshcmu** [grafana/grafana#36978] [General]
Would like to make a playlist for the same dashboard but with different time ranges. For example, "Dashboard A" with "last 15 mins" , "last 1 hour", "last 6 hours" , "last 24 hours", etc.

**Comment by jsedy7** [comment]
I need it, please! :+1:  I have a 7 dashboards in a playlist and I would like to change time on screen for specific dashboard. Thank you! :pray:

**Comment by jorioux** [comment]
I would like this also, is this feature coming soon?

**Comment by bergquist** [comment]
This is not something that the Grafana team will be working on. The feature makes sense but its too much of an edge case for us right now. If anyone wants to take a stab at building this feature I would be happy to review a PR.

**Comment by opensourcefan** [comment]
This is still open so I too will add my request for this. Honestly I'm not sure why it isn't asked for more often as each dashboard has its own data, some being easier to evaluate than others. It would give the ability to "flash" information that isn't directly relevant to monitoring as well.

**Comment by slavirotte** [comment]
I'd also like to have such a feature. I think it's essential for the playlist to be able to configure the display time for each of the dashboards in the playlist.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
